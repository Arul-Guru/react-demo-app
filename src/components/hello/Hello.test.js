// src/components/Hello.test.js
import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Hello from './Hello';

test('renders Hello component with name', () => {
  const { getByText } = render(<Hello name="World" />);
  expect(getByText('Hello, World!')).toBeInTheDocument();
});

test('matches snapshot', () => {
    const { asFragment } = render(<Hello name="World" />);
    expect(asFragment()).toMatchSnapshot();
  });
