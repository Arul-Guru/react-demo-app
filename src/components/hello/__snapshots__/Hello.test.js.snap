// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Hello, World!
  </div>
</DocumentFragment>
`;

exports[`renders correctly 1`] = `
<div>
  Hello, 
  World
  !
</div>
`;
